docs/source/_templates/layout.html

```diff
@@ -0,0 +1,48 @@
+{% extends "!layout.html" %}
+
+{% block extrahead %}
```

Comment:
you can delete everything from line 3 to 24 and 29 to 48 - those overrides were specific to ExecuTorch unless you want your left nav to work like there.